package/gluon-web-wifi-config/luasrc/lib/gluon/config-mode/model/admin/wifi-config.lua

```diff
@@ -8,7 +9,6 @@ local function txpower_list(phy)
 	local off  = tonumber(iwinfo.nl80211.txpower_offset(phy)) or 0
 	local new  = { }
 	local prev = -1
-	local _, val
```

Comment:
Loop variables are implicitly local.

Reply:
Which is why the line can be removed from what I understand. It's not directly related to the pr though.